contrib/utilities/setup_astyle.sh

```diff
@@ -19,6 +19,12 @@
 # The installed binary is used in the 'indent' script in case astyle is
 # installed by this script.
 #
+# This script only works on Linux, and also puts the 'astyle'
```

Comment:
In fact, the requirement is more Unix than Linux, isn't it? It is also supposed to work, e.g. for `macOS`. Or is this not the case? (I don't have a `macOS` available.)

Reply:
Well, good question. Further down in this file, we specifically download the file `astyle_2.04_linux.tar.gz`, which made me think that the script will really only work for linux. Indeed, the directory in question (https://sourceforge.net/projects/astyle/files/astyle/astyle%202.04/) has different files for Max and Windows.